troubleshooting/sending/gas-required-exceeds-allowance-or-always-failing-transaction.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Gas required exceeds allowance or always failing transaction
+tags:
+  - Gas
+  - Transactions
+date_published: '2020-11-17'
+date_modified: '2020-11-17'
+---
+
+This error can occur, usually when you are trying to interact with a contract. It means that MyCrypto could not estimate the gas limit for your transaction. There are multiple causes for this error, such as:
```

Comment:
This error can occur, usually when you are trying to interact with a contract. It means that MyCrypto could not estimate the gas limit for your transaction. There are multiple causes for this error, such as:

-change to-

This error can occur when interacting with a contract, and means that MyCrypto could not estimate the gas limit for your transaction.
<break line>
This error can be caused by a few reasons, including: